app/templates/components/billing-subscription.hbs

```diff
@@ -7,7 +7,7 @@
       {{#if model.subscriptions.isGithub}}
         {{#if model.trials.hasActiveTrial}}
           <p data-test-trial-message class="source plan-status-message">
-            You're trialing Travis CI via your <a href={{config.marketplaceEndpoint}} class="billing-section-link">Github Marketplace subscription</a>.
+            You're trying Travis CI via your <a href={{config.marketplaceEndpoint}} class="billing-section-link">Github Marketplace subscription</a>.
```

Comment:
I understand you're trying to say you're using a trial, but trialing isn't really a normal word.